lib/radiator/directory/collaborator.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Radiator.Directory.Collaborator do
```

Comment:
I'm not too unhappy with the name `Collaborator` but my preferred naming is `Collaboration`. Reasoning: The user/person _is_ the collaborator but this entity describes the relationship between a user and subject: a collaboration.

Reply:
To me collaboration would be work product. E.g. the result of this, or the ongoing activity. So I would leave it like this for now since you are not too unhappy. ^^